code/modules/projectiles/projectile.dm

```diff
@@ -240,330 +246,13 @@
-						|| def_zone == BP_CHEST)
-							def_zone = BP_HEAD
-						if(def_zone ==  BP_GROIN || def_zone == BP_L_LEG || def_zone == BP_R_LEG)
-							def_zone = BP_CHEST
-						if(istargetloc(target_mob) == 0)
-							def_zone = pick(BP_R_ARM, BP_L_ARM, BP_CHEST, BP_HEAD)
-						//head
-
-				if(8)
-					if(p_y <= 10) //legs level
-						//lleg
-						if(def_zone == BP_L_LEG \
-						|| def_zone == BP_GROIN || def_zone == BP_L_ARM \
-						|| def_zone == BP_CHEST)
-							def_zone = BP_L_LEG
-
-						if(def_zone == BP_HEAD || def_zone == BP_R_ARM)
-							def_zone = BP_CHEST
-
-						if(def_zone == BP_R_LEG)
-							def_zone = BP_R_LEG
-						if(istargetloc(target_mob) == 0)
-							def_zone = pick(BP_R_LEG, BP_L_LEG, BP_CHEST)
-
-					if(p_y > 10 && p_y <= 13) //groin level
-						if(p_x < 16)
-							if(def_zone == BP_L_ARM || def_zone == BP_R_LEG \
-							|| def_zone == BP_GROIN || def_zone == BP_R_ARM \
-							|| def_zone == BP_CHEST || def_zone == BP_L_LEG)
-								def_zone = BP_GROIN
-							if(def_zone == BP_HEAD)
-								def_zone = BP_CHEST
-							//groin
-						if(p_x >= 16)
-							if(def_zone == BP_L_ARM || def_zone == BP_L_LEG \
-							|| def_zone == BP_GROIN \
-							|| def_zone == BP_CHEST || def_zone == BP_HEAD)
-								def_zone = BP_L_ARM
-							if(def_zone == BP_HEAD || def_zone == BP_R_LEG)
-								def_zone = BP_CHEST
-							if(def_zone == BP_R_ARM)
-								def_zone = BP_R_ARM
-						if(istargetloc(target_mob) == 0)
-							def_zone = pick(BP_R_ARM, BP_L_ARM, BP_CHEST, BP_GROIN)
-							//left_arm
-
-					if(p_y > 13 && p_y <= 22)
-						if(p_x < 16)
-							if(def_zone == BP_L_ARM || def_zone == BP_R_LEG \
-							|| def_zone == BP_GROIN || def_zone == BP_R_ARM \
-							|| def_zone == BP_CHEST || def_zone == BP_L_LEG \
-							|| def_zone == BP_HEAD)
-								def_zone = BP_CHEST
-							//chest
-						if(p_x >= 16)
-							if(def_zone == BP_L_ARM || def_zone == BP_L_LEG \
-							|| def_zone == BP_GROIN \
-							|| def_zone == BP_CHEST || def_zone == BP_HEAD)
-								def_zone = BP_L_ARM
-							if(def_zone == BP_R_LEG || def_zone == BP_HEAD)
-								def_zone = BP_CHEST
-							if(def_zone == BP_R_ARM)
-								def_zone = BP_R_ARM
-							//larm
-						if(istargetloc(target_mob) == 0)
-							def_zone = pick(BP_R_ARM, BP_L_ARM, BP_CHEST)
-
-					if(p_y > 22 && p_y <= 32)
-						if(def_zone == BP_L_ARM \
-						|| def_zone == BP_R_ARM \
-						|| def_zone == BP_CHEST)
-							def_zone = BP_HEAD
-						if(def_zone ==  BP_GROIN || def_zone == BP_L_LEG || def_zone == BP_R_LEG)
-							def_zone = BP_CHEST
-						if(istargetloc(target_mob) == 0)
-							def_zone = pick(BP_R_ARM, BP_L_ARM, BP_CHEST, BP_HEAD)
-						//head
-
-
-
-			result = target_mob.bullet_act(src, def_zone)//this returns mob's armor_check and another - see modules/mob/living/living_defense.dm
-
+			var/spread = max(base_spreading - (spreading_step * distance), 0)
+			var/aim_hit_chance = max(0, projectile_accuracy)
+			if(prob(aim_hit_chance))
```

Comment:
Can prob receive a value > 100? (if the person has a super-high vig then aim hit chance could be > 100 with the boost frm VIG).

Reply:
Should be able to.
![yeea](https://user-images.githubusercontent.com/61743710/116099311-20cfb100-a6ac-11eb-8c5c-1d8182b5c4ec.png)